services/apis/fence/fenceTasks.js

```diff
@@ -218,7 +218,7 @@ module.exports = {
     return I.sendGetRequest(url, headers).then((res) => {
       // follow redirect back to fence
       let sessionCookie = getCookie('fence', res.headers['set-cookie']);
-      headers = { Cookie: 'dev_login=' + mockLoginUser + ';' + 'fence=' + sessionCookie };
+      headers.Cookie += `; fence=${sessionCookie}`;
```

Comment:
wait, don't we need the dev_login cookie for other tests?

Reply:
the headers already contain access_token and dev_login, the fix is appending the session cookie (before the fix, this line was removing the access_token from the headers)